src/db/mod.rs

```diff
@@ -394,11 +383,11 @@ impl Database {
     }
 
     /// Helper function to load a database into its internal XML chunks
-    pub fn get_xml_chunks(
+    pub fn get_xml(
```

Comment:
@sseemayer I like this renaming. Good idea to do those before the API becomes stable with `1.X`